components/selection/README.md

```diff
@@ -2,146 +2,328 @@
 
 The selection components (`d2l-selection-action`, `d2l-selection-input`, `d2l-selection-select-all`, `d2l-selection-summary`, `d2l-selection-action`) are low-level components that provide the ability to create selection interfaces with select-all and bulk-action behaviours.
 
+<!-- docs: start hidden content -->
 ![Selection](./screenshots/selection-multiple.png?raw=true)
 
 ![Selection](./screenshots/selection-single.png?raw=true)
+<!-- docs: end hidden content -->
 
 ## SelectionMixin
 
-The selection components above work with a component that extends the `SelectionMixin`, which acts like a controller for the checkboxes, radios, actions, etc. The `d2l-selection-input` component must be placed _within_ the component that extends the `SelectionMixin`.  The other selection components may also be placed inside the `SelectionMixin` component, or in the same DOM scope with the `selection-for` attribute set to the id of that component.
-
-The `d2l-list` already extends `SelectionMixin` and should always be used for lists, however a custom selection control can be easily defined to enable the use of these selection controls in different semantic contexts or radically different layouts. The `SelectionMixin` defines the `selection-single` attribute that consumers can specify for single selection behaviour.
-
-```javascript
-import { html, LitElement } from 'lit-element/lit-element.js';
-import { SelectionMixin } from '@brightspace-ui/core/components/selection-mixin.js';
-
-class CustomSelection extends SelectionMixin(LitElement) {
-  render() {
-    return html`
-      <slot></slot>
-    `;
-  }
-}
-customElements.define('d2l-custom-selection', CustomSelection);
-```
-
-**Properties:**
-
-| Property | Type | Description |
-|--|--|--|
-| `selection-single` | Boolean | Whether to render with single selection behaviour. If `selection-single` is specified, the nested `d2l-selection-input` elements will render radios instead of checkboxes, and the selection component will maintain a single selected item. |
-
-The selection components can then be used within the custom selection component as shown below.
+The selection components below work with a component that extends the `SelectionMixin`, which acts like a controller for the checkboxes, radios, actions, etc. The `d2l-selection-input` component must be placed _within_ the component that extends the `SelectionMixin`.  The other selection components may also be placed inside the `SelectionMixin` component, or in the same DOM scope with the `selection-for` attribute set to the id of that component.
 
+<!-- docs: demo live name:d2l-demo-selection display:block -->
```

Comment:
Similar to table and other `mixin` component examples, this uses a `d2l-demo-selection` component that does exist in order to enable viewing and interacting the the properties (in this case `selection-single`)